Reducers/Reducer.js

```diff
@@ -0,0 +1,78 @@
+//todo get inital state from storage
+//todo implement import/export
+//todo learn functional programming. //per Ben. 
+
+const initialState = {
```

Comment:
The entire state of the app lives here for now. This will eventually be broken out into separate files that all get joined together by the createStore function in App.js, but it will still act like as ingle large state object.